mypy/test/testsolve.py

```diff
@@ -2,10 +2,11 @@
 
 import typing
 
-from mypy.myunit import Suite, assert_equal
+from mypy.myunit.assertions import *
```

Comment:
Actually, it probably *would* be doable to call out to py.test for the actual assertion rewriting.

But at that point - what *exactly* is the point of writing our own unit test framework from scratch again instead of reusing someone else's?

If we did all our unit tests via pytest would could avoid the special case for mypy.codec since pytest is python2-compatible already.